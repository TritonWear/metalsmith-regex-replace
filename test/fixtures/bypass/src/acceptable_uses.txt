That's a |shitty| picture of me.
This is a picture of shit.
The weather outside is |shitty|.
My bad diarrhea made the
inside of the toilet bowl shitty,
and I had to clean it with a
rag, which then also became
shitty.