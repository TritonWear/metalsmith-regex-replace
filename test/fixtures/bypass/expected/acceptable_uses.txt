That's a shitty picture of me.
This is a picture of poop.
The weather outside is shitty.
My bad diarrhea made the
inside of the toilet bowl gross,
and I had to clean it with a
rag, which then also became
gross.